apps/condo/domains/contact/queries/Contact.graphql

```diff
@@ -50,6 +50,39 @@ query getContactsExistence ($where: ContactWhereInput) {
     }
 }
 
+query getBasicContactInfo($where: ContactWhereInput, $first: Int, $skip: Int, $sortBy: [SortContactsBy!]) {
```

Comment:
The name "basic" doesn’t make it clear what this query is for Maybe something like `getContactsForClientCard` would be better. Same goes for the employees